src/content/docs/releases/index.mdx

```diff
@@ -11,11 +11,84 @@ Adobe organizes storefront application releases into logical suite versions and
 
 The following sections provide high-level summaries of each release suite and details about the compatibility between storefront components. To see more details about each drop-in component by version, go to the [changelog](/releases/changelog/).
 
-### June 2025 suite
+## August 2025 suite
+
+This suite release delivers significant enhancements to existing drop-in components, providing improved functionality and better user experiences for your storefront. The following table lists notable highlights. To see more details about each drop-in component by version, go to the [changelog](/releases/changelog/).
+
+### Updated boilerplate
+
+The Commerce boilerplate has been significantly enhanced with new carousel and cards-list components from Adobe Store, improved Storefront Builder compatibility, and comprehensive architecture improvements. Key highlights include major scripts.js refactoring for better maintainability, enhanced drop-in functionality across Cart, Checkout, and Wishlist components, optimized header performance, and improved accessibility with global placeholders and modal localization. Additional updates include reCAPTCHA enhancements, Product Discovery search fixes, comprehensive documentation improvements, and removal of deprecated widgets for cleaner implementation.
```

Comment:
Are we still using "Storefront Builder"? I thought we scrubbed it? Maybe I'm misremembering.

Reply:
Good catch. I'll change it to Commerce Storefront.